packages/react-components/lib/navigation-bar.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Is this still being used?

Reply:
nope, but I was thinking to just leave it if it would be useful for building other dashboards? can remove if you think it's cluttering the storybook!